chesster.js

```diff
@@ -300,14 +301,13 @@ function sayCaptains(self, convo, callback){
 /* rating */
 
 controller.hears([
-	'rating'
+    players.appendPlayerRegex("rating", true)	
```

Comment:
with this change I think we can remove the error handling below.

it will no longer match if there is no player